core/source/graphics/render_gl.cpp

```diff
@@ -580,18 +580,19 @@ class Render_graphics_opengl_threejs : public Render_graphics_opengl_vertex_buff
 	std::string **output_string;
 	int morphVertices, morphColours, morphNormals, numberOfResources;
 	char **filenames;
-
+	int isInline;
```

Comment:
In future should be:
bool isInline;
Same in several other places including functions/constructors.